### PR TITLE
basic scenarios implemented

### PR DESCRIPTION
**Pull request by EnitanOO**
[AB#9453](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9453)
[AB#13859](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13859)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature [hunk 1]
```diff
@@ -437,3 +437,5 @@ Scenario: Catalogue Solutions - Edit Service Recipients - Go Back
 	When the User chooses to edit the service recipients
 	And the User chooses to go back
 	Then the Edit Price form displays the same number of recipients as earlier
+
+
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature:441]: Don't need 2 additional empty lines here

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServicesEditRecipients.feature [hunk 2]
```diff
@@ -41,3 +41,47 @@ Scenario: Additional Services - Enabled Delete Button shows correct Text
...
+
+Scenario: Additional Services - Go Back
+   Given A User has saved the Additional Service to the order
+    And the User has chosen to manage the Additional Services section
+    And the User chooses to edit service recipients
+    Then they are presented with the select Service Recipient form
+    When they choose to go back
+    Then the Edit Price form displays the same number of recipients as earlier 
+
+
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServicesEditRecipients.feature:87]: Don't need 2 additional empty lines here

File: src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs [hunk 3]
```diff
@@ -971,6 +971,25 @@ public void ClickUnselectedCheckbox()
...
+            foreach (var checkbox in checkboxes)
+            {
+                if (bool.Parse(checkbox.GetProperty("checked")))
+                {
+                    checkbox.Click();
+
+                    if (!bool.Parse(checkbox.GetProperty("checked")))
+                    {
+                        Wait.Until(d => d.FindElements(Objects.Pages.Common.Checkbox).Where(s => bool.Parse(s.GetProperty("checked"))).Count() != checkedCheckboxes + 1);
+                        break;
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs:987]: Not sure I understand what you're trying to do here. Can you elaborate?

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature (hunk 1 above)
Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/CatalogueSolutions.feature:441]: will sort

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServicesEditRecipients.feature (hunk 2 above)
Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServicesEditRecipients.feature:87]: will sort

File: src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs (hunk 3 above)
Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs:987]: ...checked. I don't really know what I am doing with that as I don't need it. 
 I have removed.

Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs:987]: All three implemented.

Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs:987]: checking through again 
Can you see anything else amiss?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs:987]: Not from my look through I didn't

Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs]: sent back

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**